Makefile

```diff
@@ -0,0 +1,30 @@
+.DEFAULT_GOAL := install
+
+.PHONY: check
+check:
+	@echo Checking...
+	@git diff -s --exit-code . || (echo "Build failed: one of more files aren't staged to be committed" && exit 1)
+
+.PHONY: clean
+clean:
+	@echo Cleaning...
+	rm -rf environments/kubernetes/manifests
+	rm -rf environments/openshift/manifests
+	rm -rf environments/sre/servicemonitors
+	rm -rf environments/sre/prometheusrules
+	rm -rf environments/sre/grafana
+
+.PHONY: build
+build:
+	@echo Building...
+	@./build.sh > build.log 2>&1 
```

Comment:
If we really want a Makefile, we should get rid of the build.sh entirely.

Reply:
In general, I personally prefer to wrap more complex tasks in shell scripts, as there are things that are insane to accomplish in Makefiles. I think I'll end up removing the `Makefile` for this PR, as it's really not within the scope of the change.